packages/fxa-auth-server/config/index.ts

```diff
@@ -62,7 +62,7 @@ const conf = convict({
   geodb: {
     dbPath: {
       doc: 'Path to the maxmind database file',
-      default: path.resolve(__dirname, '../../fxa-geodb/db/cities-db.mmdb'),
+      default: require.resolve('fxa-geodb/db/cities-db.mmdb'),
```

Comment:
smooths out a bump of running under the `dist` folder, though this wouldn't affect prod anyway